### PR TITLE
Instantiate quantizer specified only by name

### PR DESCRIPTION
**Pull request by vloncar**
Instantiate quantizer without arguments if only specified by name (i.e., treat `binary` as `binart()`). Test case included

### CODE REVIEW

File: tests/safe_eval_test.py [hunk 1]
```diff
@@ -76,6 +76,9 @@ def test_safe_eval4():
   assert safe_eval("myadd2(a= 39)", globals(), b=3) == -42
   assert safe_eval("myadd2(a= 39, b = 3)", globals()) == -42
 
+def test_safe_eval5():
+  assert safe_eval("myadd", globals()) == 42
+  assert safe_eval("myadd()", globals()) == 42
```
Comment (zhuangh) [inline tests/safe_eval_test.py]: you might need to add a test case like you mentioned in #10, like in the model layer, use binary or ternary for quantizers.

thanks.

Comment (zhuangh) [reply tests/safe_eval_test.py]: hi @vloncar  does my comment make sense?

Comment (vloncar) [reply tests/safe_eval_test.py]: I made the change so that the feature is used in `qconvolutional_test.py`. Is that what you had in mind?

Comment (zhuangh) [reply tests/safe_eval_test.py]: yes! thanks.

**Pull request merged**